src/components/new-request/search-components/numbered-company-bullets.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-col id="numbered-company-bullets" :cols="cols" :md="md">
+    <template v-if="isConversion && !isAlterOnline(getConversionType)">
```

Comment:
Here's another special case, where this component always displays but, depending on the conditionals, some code block or another is displayed.

My point is that logic on whether to display a component belongs in the parent, but within a component there can be some special cases.